sound/netcat.py

```diff
@@ -0,0 +1,48 @@
+#
+# netcat.py - Quick-n-dirty network interface
+# Copyright (C) 2015  Linas Vepstas
```

Comment:
we do not need two copies of exactly the same code in this directory.  Please use the pre-existing version of netcat.